README.md

```diff
@@ -38,9 +38,14 @@ In addition to that, you may enjoy following:
 
 Contributing
 ------------
-Contributions are *very* welcome!
+**Contributions are *very* welcome!**
 
-If you see an issue that you'd like to see fixed, the best way to make it happen is to help out by submitting a pull request implementing it.
+If you see an issue that you'd like to see fixed, or want to shape out some ideas,
+the best way to make it happen is to help out by submitting a pull request implementing it.
+We welcome contributions from all, even you are not yet familiar with this project,
+We are happy to get you started, and will guide you through the process once you've submitted your PR.
```

Comment:
We definitely _want_ to do this, though we can't always promise to respond (quickly)... I think this is OK :+1: